frontend/src/App.vue

```diff
@@ -42,7 +42,7 @@ export default {
       }
     }
   },
-  destroyed() {
+  unmounted() {
```

Comment:
According to vue2 docs, there are no lifecycle hooks `unmounted` or `beforeUnmount`. Did you check if this code is still called?

https://v2.vuejs.org/v2/guide/instance#Lifecycle-Diagram
https://v2.vuejs.org/v2/api/#Options-Lifecycle-Hooks

Reply:
i reverted that change.
They were not called